app/controllers/v0/coronavirus_chatbot/tokens_controller.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module V0
+  module CoronavirusChatbot
+    class TokensController < ApplicationController
+      skip_before_action :authenticate, only: [:create]
+
+      rescue_from 'V0::CoronavirusChatbot::TokensController::ServiceException', with: :service_exception_handler
+      rescue_from Net::HTTPError, with: :service_exception_handler
+
+      def create
```

Comment:
what sort of authentication is on this endpoint? Can anyone hit it or only va.gov front end?

Reply:
per line 6  in this same file, none